build.gradle

```diff
@@ -2,10 +2,8 @@
 
```

Comment:
Could we add a comment with a link to the file in `a-s` that this is based on?  That should make it a little easier for the next person to come along and do this.